### PR TITLE
bindmount for sf* and c3

### PR DESCRIPTION
**Pull request by plandolt**
PR for issue #11 

I added the local mounts for sf\* and contao3. Another one who is using the other application might add the mounts later.

### CODE REVIEW

**Comment by dominikzogg** [comment]
@scuben looks great

File: Vagrantfile [hunk 1]
```diff
@@ -154,4 +154,8 @@ Vagrant.configure(2) do |config|
         sh.path = 'ansible/ansible-on-guest.sh'
         sh.args = ['ansible/playbook.yml', setupConfig.to_json.split(' ').join('\u0020')]
     end
+
+    config.vm.provision 'shell', run: "always" do |sh|
```
Comment (dominikzogg) [inline Vagrantfile]: Missing check if application is null

Comment (plandolt) [inline Vagrantfile]: ah yes, will fix that

**Comment by plandolt** [comment]
does not break on null application.

**Comment by dominikzogg** [comment]
@scuben thank you

would be great if you would change `/tmp/cache/` to `/tmp/symfony/var/` to be sure everything is within a directory that its easy to find, and no other access it

**Comment by dominikzogg** [comment]
@scuben tested it with symfony 3 allready, great idea!!!

**Comment by plandolt** [comment]
@dominikzogg yeah, it speeds up my app too :) Added the application to the temp dir given as an argument.

**Comment by dominikzogg** [comment]
@scuben, thank you for your contribution

**Pull request merged**